app/webpacker/components/RegistrationsV2/Register/RegistrationOverview.jsx

```diff
@@ -63,15 +113,24 @@ export default function RegistrationOverview({
             </label>
             {registration.guests}
           </FormField>
-          { editsAllowed && (
+          <ButtonGroup widths={2}>
```

Comment:
What happens when `deleteAllowed` is false, but the button group width is still set to `2`?

Reply:
let me see

Reply:
still works